Sources/Views/MessageInputBar.swift

```diff
@@ -437,7 +447,10 @@ open class MessageInputBar: UIView {
                 inputTextView.invalidateIntrinsicContentSize()
             }
         }
-        return CGSize(width: bounds.width, height: heightToFit)
+        let totalPadding = padding.top + padding.bottom + topStackViewPadding.top + textViewPadding.top + textViewPadding.bottom
```

Comment:
Do we need to consider `topStackViewPadding.bottom`? 

What about the other stack views? 
Can something like this happen where the left or right stackviews increases the size?
```
[            Top            ]
| L |                       
|   |    [TextView]     [ R ]
[          Bottom           ]
```

Reply:
Ya i wanna screenhero about this when you have a moment

Reply:
If extra height is needed it will be given because the `MessageInputBar` autoresizingMask = flexibleHeight